collect_app/src/main/java/org/odk/collect/android/preferences/PreferenceKeys.java

```diff
@@ -91,12 +92,12 @@
         hashMap.put(KEY_GOOGLE_SHEETS_URL,          "");
         hashMap.put(KEY_FORMLIST_URL,               Collect.getInstance().getString(R.string.default_odk_formlist));
         hashMap.put(KEY_SUBMISSION_URL,             Collect.getInstance().getString(R.string.default_odk_submission));
-        hashMap.put(KEY_NAVIGATION,                 "swipe");
-        hashMap.put(KEY_CONSTRAINT_BEHAVIOR,        "on_swipe");
+        hashMap.put(KEY_NAVIGATION,                 NAVIGATION_SWIPE);
+        hashMap.put(KEY_CONSTRAINT_BEHAVIOR,        CONSTRAINT_BEHAVIOR_ON_SWIPE);
         hashMap.put(KEY_COMPLETED_DEFAULT,          true);
-        hashMap.put(KEY_MAP_SDK,                    "google_maps");
-        hashMap.put(KEY_MAP_BASEMAP,                "streets");
-        hashMap.put(KEY_AUTOSEND,                   "off");
+        hashMap.put(KEY_MAP_SDK,                    GOOGLE_MAPS);
+        hashMap.put(KEY_MAP_BASEMAP,                GOOGLE_MAPS_BASEMAP_DEFAULT);
+        hashMap.put(KEY_AUTOSEND,                   Collect.getInstance().getString(R.string.off_untranslated));
```

Comment:
I think that to match the others, this should be something like `AUTOSEND_OFF` defined above.

Reply:
👍 will do